tests/test_inputlocationmessagecontent.py

```diff
@@ -19,46 +19,22 @@
 
 import pytest
 
-from telegram import InputMessageContent, InputLocationMessageContent
-
-
-@pytest.fixture(scope='function')
-def json_dict():
-    return {
-        'longitude': TestInputLocationMessageContent.longitude,
-        'latitude': TestInputLocationMessageContent.latitude,
-    }
+from telegram import InputLocationMessageContent
 
 
 @pytest.fixture(scope='class')
 def input_location_message_content():
-    return InputLocationMessageContent(TestInputLocationMessageContent.longitude,
-                                       TestInputLocationMessageContent.latitude)
+    return InputLocationMessageContent(longitude=TestInputLocationMessageContent.longitude,
```

Comment:
Why did you add these as kwargs?
Throughout the rest of the test we pass as an arg if it's a arg and kwarg if kwarg.